csharp/src/Ice/PropertiesI.cs

```diff
@@ -89,6 +106,12 @@ public string[] getPropertyAsList(string key)
         return getPropertyAsListWithDefault(key, null);
     }
 
+    public string[] getIcePropertyAsList(string key)
+    {
+        string[] defaultList = IceUtilInternal.StringUtil.splitString(getDefaultProperty(key), ", \t\r\n");
```

Comment:
We should use https://learn.microsoft.com/en-us/dotnet/api/system.string.split?view=net-8.0

Reply:
This is not the same as our splitSting deals with "quoted" properties.